spaces/S000065/properties/P000204.md

```diff
@@ -0,0 +1,9 @@
+---
+space: S000065
+property: P000204
+value: true
+---
+
+The space is connected, and the point $\frac{1}{2}$ is a cut point,
+since $[0, \frac{1}{2})$ and $(\frac{1}{2}, 1] \cup 1^*$ is a pair of disjoint
```

Comment:
```suggestion
The space is {P36} (since {S65|P199}),
and the point $\frac{1}{2}$ is a cut point,
since $[0, \frac{1}{2})$ and $(\frac{1}{2}, 1] \cup \{1^*\}$ is a pair of disjoint
```